### PR TITLE
S02US27_delete_activity

### PR DESCRIPTION
**Pull request by cedriclv**
first commit

### CODE REVIEW

**Review by Planger45 — APPROVED** [review]

**Review by DrissB45 — APPROVED** [review]

File: src/View/Activity/index.html.twig [hunk 1]
```diff
@@ -18,11 +18,10 @@
...
                                         <a href="#">Editer</a>
                                     </div>
-                                    <form action="" method="post">
-                                    <div class="btn btn-danger">
-                                        <a href="#">Supprimer</a>
-                                    </div>
-                                    </form>
+                                        <form action="/activity/delete" method="POST">
+                                            <input type="hidden" name="id" value="{{ activity.id }}">
+                                            <button>Delete</button>
```
Comment (sblondeau) [inline src/View/Activity/index.html.twig:23]: supprimer

File: src/Controller/ActivityController.php [hunk 2]
```diff
@@ -90,4 +90,15 @@ public function add()
...
     }
+
+    public function delete()
+    {
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            $id = trim($_POST['id']);
+            $activityManager = new ActivityManager();
+            $activityManager->delete((int)$id);
+
+            Header('Location:/activity');
```
Comment (sblondeau) [inline src/Controller/ActivityController.php:101]: header (minuscule)

**Review by sblondeau — CHANGES_REQUESTED** [review]

File: src/View/Activity/index.html.twig (hunk 1 above)
Comment (cedriclv) [reply src/View/Activity/index.html.twig:23]: done

File: src/View/Activity/index.html.twig [hunk 3]
```diff
@@ -18,11 +18,10 @@
                                     <div class="btn btn-primary">
                                         <a href="#">Editer</a>
```
[the remark this answers is not in the document]
Comment (cedriclv) [reply src/View/Activity/index.html.twig:19]: done

**Review by DrissB45 — APPROVED** [review]

File: src/Controller/ActivityController.php (hunk 2 above)
Comment (cedriclv) [reply src/Controller/ActivityController.php]: done

**Pull request merged**